config/initializers/cors.rb

```diff
@@ -0,0 +1,16 @@
+# Be sure to restart your server when you modify this file.
+
+# Avoid CORS issues when API is called from the frontend app.
+# Handle Cross-Origin Resource Sharing (CORS) in order to accept cross-origin AJAX requests.
+
+# Read more: https://github.com/cyu/rack-cors
+
+# Rails.application.config.middleware.insert_before 0, Rack::Cors do
+#   allow do
+#     origins "example.com"
+#
+#     resource "*",
+#       headers: :any,
+#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
+#   end
+# end
```

Comment:
For the API, CORS is important. It could be that we do CORS setup elsewhere, in that case we need to move it here, or document that we are doing it elsewhere.

Reply:
i.e. it would be nice if running a browser-based app that uses the Foodsoft API would remain working.

Reply:
thanks for your comment! I moved it from config/application.rb to the correct initializer and did a quick test with the foodsoft-shop application